### PR TITLE
Improved consistency of external APIs

### PR DESCRIPTION
**Pull request by jalextowle**
- Implements an `ApolloLink` subclass that allows browser nodes to support most GraphQL endpoints. Subscriptions and raw queries are excluded from this PR.
- Change `GetOrders` to `FindOrders` in several files.

## Small Changes
- Fixed a bug in the integration tests that would cause us not to fail tests due to browser errors.

### CODE REVIEW

File: packages/mesh-browser-lite/src/mesh.ts [hunk 1]
```diff
@@ -151,8 +151,8 @@ export class Mesh {
      */
     public onError(handler: (err: Error) => void): void {
         this._errHandler = handler;
-        if (this._wrapper !== undefined) {
-            this._wrapper.onError(this._errHandler);
+        if (this.wrapper !== undefined) {
```
Comment (z2trillion) [inline packages/mesh-browser-lite/src/mesh.ts:154]: can this be shortened to `this.wrapper?.onError(this._errHandler)`?

**Review by z2trillion — APPROVED** [review]
I know this usage predates this PR, but on file lines that you touch, can you use the `?` operator instead of `if x !== undefined ...`?

File: graphql/generated/generated.go [hunk 2]
```diff
@@ -3945,6 +3954,40 @@ func (ec *executionContext) _Stats_ethRPCRateLimitExpiredRequests(ctx context.Co
...
+	}
+	if resTmp == nil {
+		if !graphql.HasFieldError(ctx, fc) {
+			ec.Errorf(ctx, "must not be null")
+		}
+		return graphql.Null
+	}
+	res := resTmp.([]string)
+	fc.Result = res
+	return ec.marshalNString2ᚕstringᚄ(ctx, field.Selections, res)
```
Comment (Recmo) [inline graphql/generated/generated.go:3988]: Strange unicode characters here, but the same happens in some other functions, so not caused by this PR. Still, what's going on here?

**Review by Recmo — APPROVED** [review]
LGTM!

File: graphql/generated/generated.go (hunk 2 above)
Comment (jalextowle) [reply graphql/generated/generated.go:3988]: It's part of graphql generation. I'm not completely sure what these replacements mean and why unicode is being used, but I did find an instance where they are using these unicode characters during codegen: https://github.com/99designs/gqlgen/blob/f4669ba9b54cdda6340c6e0c16d05eff2ee4fa21/codegen/templates/templates.go#L232.

**Pull request merged**